homeassistant/components/workday/__init__.py

```diff
@@ -73,32 +91,199 @@ async def _async_validate_country_and_province(
         ) from ex
 
 
-async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+def _get_obj_holidays(
```

Comment:
I know it's only used once - but I think it would still sit better in the new util module